apps/entity-service/src/entities/processors/nursery.processor.ts

```diff
@@ -103,20 +104,22 @@ export class NurseryProcessor extends EntityProcessor<Nursery, NurseryLightDto,
   }
 
   async addLightDto(document: DocumentBuilder, nursery: Nursery): Promise<void> {
-    const nuseryId = nursery.id;
-
-    const nurseriesReports = await NurseryReport.nurseries([nuseryId]).findAll({
-      attributes: ["id", "seedlingsYoungTrees"]
-    });
-
-    const seedlingsGrownCount = nurseriesReports.reduce((sum, { seedlingsYoungTrees }) => {
-      return sum + (seedlingsYoungTrees ?? 0);
-    }, 0);
+    const nurseryId = nursery.id;
+
+    const seedlingsGrownCount =
+      (
+        await NurseryReport.nurseries([nurseryId])
+          .approved()
+          .findAll({
+            raw: true,
+            attributes: [[fn("SUM", col("seedlings_young_trees")), "seedlingsYoungTrees"]]
+          })
+      )[0].seedlingsYoungTrees ?? 0;
```

Comment:
Since this count is used in both DTOs, please move the code to a private method in the processor to avoid duplication.